src/main/java/com/country/name/NameController.java

```diff
@@ -16,5 +15,11 @@ public NameController(NameMapper nameMapper) {
     public List<Countries> findByNames(NameSearchRequest request) {
         return nameMapper.findByNameStartingWith(request.getStartsWith(), request.getEndsWith(), request.getContains());
     }
+    @GetMapping("/country")
```

Comment:
RESTfulにつくるならばURLのパスは複数形が望ましいです。
```suggestion
    @GetMapping("/countries")
```